express/src/config/session-storage.ts

```diff
@@ -10,9 +10,12 @@ function getSessionStorage(): RequestHandler {
     if (sessionStorage.tableName) {
         console.log("Using DynamoDB session storage");
         const DynamoDBStore = connect<SessionData>(session);
+        console.log(`********* Retrieving Session Storage. Secret is "${sessionSecret}". *********`);
```

Comment:
```suggestion
```
We do not want to log a secret....

Reply:
Sure; this was a debugging statement that I should have removed.